### ISSUE TITLE
Error with Features showing up in languages

### ISSUE THREAD

**Issue by gockleyd** [jbhaywood/5e-statblock-importer#31]
I am trying to be import monsters from the MCDM Flee Mortals preview

When importing the below Goblin the feature "Crafty" does not import as a separate feature and the text shows under the languages header

Goblin Boss
Small Humanoid (Goblin), Any Alignment
Armor Class 17 (studded leather armor, shield)
Hit Points 36 (8d6 + 8)
Speed 30 ft., climb 20 ft.
STR DEX CON INT WIS CHA
10 (+0) 16 (+3) 13 (+1) 12 (+1) 12 (+1) 10 (+0)
Saves Dex +5, Wis +3
Skills Insight +3, Intimidation +2, Stealth +5
Senses darkvision 60 ft., passive Perception 11
Languages Common, Goblin
Proficiency Bonus +2
Crafty. The boss doesn’t provoke opportunity attacks
when they move out of an enemy’s reach.
ACTIONS
Multiattack. The boss makes two Shortsword or Shortbow
attacks. They can use Command in place of one attack.
Shortsword. Melee Weapon Attack: +5 to hit, reach 5 ft.,
one target. Hit: 6 (1d6 + 3) piercing damage.
Shortbow. Ranged Weapon Attack: +5 to hit, range 80/320 ft.,
one target. Hit: 6 (1d6 + 3) piercing damage.
Command. The boss chooses one ally they can see within 30 feet
of them. If the target can hear the boss, the target can use their
reaction to move up to their speed or make one weapon attack.
BONUS ACTIONS
Get Reckless (Recharge 6). Each willing ally within 30 feet of the
boss that can hear them becomes reckless until the start of the
boss’s next turn. While reckless, a creature has advantage on attack
rolls, and attack rolls against the creature have advantage.
REACTIONS
Cowardly Commander. When a creature the boss
can see hits them with an attack, the boss chooses
a willing ally within 5 feet of them. The attack
hits the ally instead.

**Comment by Greyhart42** [comment]
Yup. I've got the same exact problem, importing from Flee Mortals.

I also notice that while it imports Common as a language, Orc is imported into Special (Split with Semi Colon) right in front of Crafty. (I've already moved Orc to its proper place in the images, but they show where the content is being placed.

![image](https://user-images.githubusercontent.com/47504205/184501727-83b8c394-4eb4-4581-96f0-1123353ccb63.png)
![image](https://user-images.githubusercontent.com/47504205/184501736-f3f8b472-046c-40d4-a722-eed673da07f3.png)
